src/service/data_products/genome_attributes.py

```diff
@@ -104,8 +104,8 @@ async def get_ranks(
     r: Request,
     collection_id: str = PATH_VALIDATOR_COLLECTION_ID,
     sort_on: str = Query(
-        default=names.FLD_GENOME_ATTRIBS_GENOME_NAME,
-        example="genome_size",
+        default=names.FLD_GENOME_ATTRIBS_ACCESSION,
+        example="accession",
```

Comment:
No need to change the example, it was different from the default before

Reply:
👍

Reply:
:+1: